docs/advanced_topics/customisation/admin_templates.rst

```diff
@@ -91,6 +91,21 @@ To replace the welcome message on the dashboard, create a template file ``dashbo
 
     {% block branding_welcome %}Welcome to Frank's Site{% endblock %}
 
+.. _custom_user_interface_fonts:
+
+Custom user interface fonts
+===========================
+
+To customise the font families used in the admin user interface, inject a CSS file using the hook :ref:`insert_global_admin_css` and override the variables within the ``:root`` selector:
+
+
+.. code-block:: text
```

Comment:
Not sure why we’ve been using `text` in this file so far, there is CSS too, unless that renders badly for `:root` for some reason?

```suggestion
.. code-block:: css
```